ui/tests/acceptance/task-group-detail-test.js

```diff
@@ -441,4 +454,31 @@ module('Acceptance | task group detail', function(hooks) {
       }
     });
   });
+
+  test('when a task group has at least count scaling events and the count scaling events outnumber the non-count scaling events, a timeline is shown instead of an accordion', async function(assert) {
```

Comment:
```suggestion
  test('when a task group has at least two count scaling events and the count scaling events outnumber the non-count scaling events, a timeline is shown instead of an accordion', async function(assert) {
```